HTML/html_completions.py

```diff
@@ -588,7 +602,7 @@ def attribute_completions(self, view, pt, prefix):
             [
                 sublime.CompletionItem(
                     trigger=attr,
-                    completion=f'{attr}="$1"{suffix}',
+                    completion=self.generate_attribute(attr, suffix),
```

Comment:
With the `is_bool()` it was possible to avoid one level of function calls, which also slightly improves performance.

```suggestion
                    completion=f'{attr}{suffix}' if is_bool(attr) else f'{attr}="$1"{suffix}',
```

Reply:
It might not be a convention in this file but creating a top-level Set variable would also be more performant (no need to call a function and recreate the set on each call).

Reply:
I haven't seen any timing improvements when using global set of boolean attributes. Maybe my CPU is to fast and set to small.

Everyone interested can test it by setting the global `ENABLE_TIMING = True`.

Reply:
I guess it won't really make any noticeable difference for this case unless this code is triggered a 10000 times a second or something.

Still, avoiding an extra function call and an initialization of Set should be faster (in general).